conformance/src/main/scala/za/co/absa/enceladus/conformance/datasource/DataSource.scala

```diff
@@ -61,4 +67,17 @@ object DataSource {
   private[conformance] def setData(path: String, data: Dataset[Row]) {
     dfs += (path -> data)
   }
+
+  private def loadMappingTable(path: String)
+                              (implicit spark: SparkSession): DataFrame = {
+    try {
+      spark.read.parquet(path)
+    } catch {
+      case ex: AnalysisException if ex.getMessage.contains("Unable to infer schema for Parquet") =>
```

Comment:
Why check the message? This would make us dependent on the message not changing. 
In what other case would an `AnalysisException` by thrown by `spark.read.parquet`?

Reply:
I see that `path does not exist` is another case for `AnalysisException`. Alright, let's stick with this. The tests will start failing if we move to a different spark version with a different message and we can adjust then.